rpc/did/runtime-api/src/lib.rs

```diff
@@ -86,14 +89,65 @@ pub struct DidLinkedInfo<
 ///
 /// This will be returned by the runtime and processed by the client side RPC
 /// implementation.
-pub type RawDidLinkedInfo<DidIdentifier, AccountId, Balance, Key, BlockNumber> =
-	DidLinkedInfo<DidIdentifier, AccountId, Vec<u8>, Vec<u8>, Vec<u8>, Vec<u8>, Balance, Key, BlockNumber>;
+pub type RawDidLinkedInfo<DidIdentifier, AccountId, LinkableAccountId, Balance, Key, BlockNumber> = DidLinkedInfo<
+	DidIdentifier,
+	AccountId,
+	LinkableAccountId,
+	Vec<u8>,
+	Vec<u8>,
+	Vec<u8>,
+	Vec<u8>,
+	Balance,
+	Key,
+	BlockNumber,
+>;
 
 sp_api::decl_runtime_apis! {
-	/// The API to query account nonce (aka transaction index).
-	pub trait DidApi<DidIdentifier, AccountId, Balance, Key: Ord, BlockNumber> where
+	// TODO: Remove this runtime API when the SDK team agrees that it is time to introduce a breaking change.
+	#[api_version(2)]
+	/// The API to query DID information.
+	pub trait DidApi<DidIdentifier, AccountId, LinkableAccountId, Balance, Key: Ord, BlockNumber> where
+		DidIdentifier: Codec,
+		LinkableAccountId: Codec,
+		AccountId: Codec,
+		BlockNumber: Codec + MaxEncodedLen,
+		Key: Codec,
+		Balance: Codec,
+	{
+		/// Given a web3name this returns:
+		/// * the DID
+		/// * public keys stored for the did
+		/// * the web3name (optional)
+		/// * associated accounts
+		/// * service endpoints
+		#[changed_in(2)]
```

Comment:
We don't need API versioning either, since we don't have Ethereum linking support yet.